packages/relay-compiler/index.d.ts

```diff
@@ -0,0 +1,366 @@
+   * in the future. Enabling this means you will have to update your application whenever the GraphQL server schema adds 
+   * new enum values to prevent it from breaking.
+   * 
+   * @default false
+   */
+  noFutureProofEnums?: boolean;
+  /** Mappings from custom scalars in your schema to built-in GraphQL types, for type emission purposes. */
+  customScalars?: RelayConfigCustomScalars;
+  /**
+   * This option enables emitting ES modules artifacts.
+   * 
+   * @default false
+   */
+  eagerEsModules?: boolean;
+  /** Relay supports two versions of the config. */
+  persistConfig?: RelayConfigPersist;
+  /** Name of the command that runs the relay compiler */
+  codegenCommand?: string;
+  /**
+   * Name of the global variable for dev mode
+   * 
+   * @default "__DEV__"
+   */
+  isDevVariableName?: string;
+  /** 
+   * Formatting style for generated files.
+   * 
+   * @default "commonjs"
+   */
+  jsModuleFormat?: 'commonjs' | 'haste';
+  /** 
+   * Options for configuring the output of compiler diagnostics. 
+   * 
+   * @default
+   * ```js
+   * { 
+   *   criticalLevel: "error",
+   * }
+   * ```
+   */
+  diagnosticReportConfig?: RelayConfigDiagnosticReportConfig;
+  featureFlags?: RelayFeatureFlags;
+  /** 
+   * Keep the previous compiler behavior by outputting an union of the raw type and null, 
+   * and not the **correct** behavior of an union with the raw type, null and undefined.
+   * 
+   * @default false
+   */
+  typescript_exclude_undefined_from_nullable_union?: boolean;
+}
+
+/**
+ * This is the configuration containing the object for multiproject configuration.
+ */
+export type RelayConfigMultiProject = {
+  /** Name of this configuration. */
+  name?: string;
+  /** Root directory relative of this configuration. */
+  root?: string;
+  /** A header to be prepend to each generated artifact. */
+  header?: string[];
+  /**
+   * A mapping directory from directory paths (relative to the root) to a project.
+   * If a path is a subdirectory of another path, the more specific path wins.
+   */
+  sources: Record<string, string>;
+  /** Name of the command that runs the relay compiler */
+  codegenCommand?: string;
+  /**
+   * Directory to be excluded under each project.
+   * 
+   * @default ["/node_modules/", "/mocks/", "/generated/"]
+   */
+  excludes?: string[];
+  /** Configuration of each project that will be reached by the compiler. */
+  projects: Record<string, RelayConfigMultiProjectItem>;
+  savedStateConfig?: {
+    mergebase?: string;
+    "mergebase-with"?: string;
+    "saved-state"?: {
+      storage?: string;
+      "commit-id"?: string;
+      config?: any;
+    };
+  };
+  /**
+   * Name of the global variable for dev mode
+   * 
+   * @default "__DEV__"
+   */
+  isDevVariableName?: string;
+}
+
+type RelayConfigMultiProjectItemSchemaConfig =
+  | { 
+      /** The path of your schema file. */
+      schema: string 
+    };
```

Comment:
I don't think a semicolon is valid here.

Reply:
I think that this isn't a problem, but I removed it to follow the pattern